Yoast/javascript
# Remove word form building functionality for English

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Removes obsolete functionality to build English word forms.

## Relevant technical choices:

* This PR removes rules that were created to be able to generate forms for the English language. With the new morphology research, this functionality is not needed anymore. For more context see https://github.com/Yoast/javascript/pull/510.

## Test instructions

This PR can be tested by following these steps:
Make sure to test together with the sister branch from premium configuration (introduced in https://github.com/Yoast/YoastSEO.js-premium-configuration/pull/100). 
* Run `yarn test` and make sure that all specs still pass.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes  https://yoast.atlassian.net/browse/LIN-255
